CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_ad_reporting v1.5.0
+
+## 🚨 Breaking Changes 🚨
+[PR #93](https://github.com/fivetran/dbt_ad_reporting/pull/93) includes the following changes based on the underlying individual package upgrades:
+
+- Pinterest Ads dependency upgraded to v0.8.0 ([PR #26](https://github.com/fivetran/dbt_pinterest/pull/26))
```

Comment:
Small suggestion to clearly call out the upstream changes that are included in the upgraded version of Pinterest Ads.

```suggestion
- Pinterest Ads dependency upgraded to v0.8.0. Please see the respective upstream [Pinterest Ads](https://github.com/fivetran/dbt_pinterest/releases/tag/v0.8.0) and [Pinterest Ads Source](https://github.com/fivetran/dbt_pinterest_source/releases/tag/v0.8.0) releases for more details.
```